### PR TITLE
 📖 Add release-0.3 branch test triggers in README

### PR DESCRIPTION
**Pull request by kashifest**
as the title says

### CODE REVIEW

**Comment by kashifest** [comment]
/override test-centos-e2e-integration-main
/override test-ubuntu-integration-main 
Its just a README addition, nothing that integration test would actually need to test.

File: README.md [hunk 1]
```diff
@@ -37,18 +37,27 @@ Eventually these tests will be replaced with BMO's own
...
 * **/test-ubuntu-integration-main** runs ansible integration tests with CAPM3
   main branch and BMO **main** branch on Ubuntu
 
-Release-0.4 branch:
+**Release-0.4 branch:**
 
 * **/test-centos-e2e-integration-release-1-5** runs CAPM3 e2e integration tests
   with CAPM3 release-1.5 branch and BMO **release-0.4 branch** on Centos
 * **/test-ubuntu-integration-release-1-5** runs ansible integration tests with CAPM3
   release-1.5 branch and BMO **release-0.4 branch** on Ubuntu
```
Comment (tuminoid) [inline README.md:52]:
```suggestion
  release-1.5 branch and BMO **release-0.4** branch on Ubuntu
```

File: README.md [hunk 2]
```diff
@@ -37,18 +37,27 @@ Eventually these tests will be replaced with BMO's own
...
 * **/test-centos-e2e-integration-main** runs CAPM3 e2e integration tests with
   CAPM3 main branch and BMO **main** branch on Centos
 * **/test-ubuntu-integration-main** runs ansible integration tests with CAPM3
   main branch and BMO **main** branch on Ubuntu
 
-Release-0.4 branch:
+**Release-0.4 branch:**
 
 * **/test-centos-e2e-integration-release-1-5** runs CAPM3 e2e integration tests
   with CAPM3 release-1.5 branch and BMO **release-0.4 branch** on Centos
```
Comment (tuminoid) [inline README.md:50]:
```suggestion
  with CAPM3 release-1.5 branch and BMO **release-0.4** branch on Centos
```

File: README.md [hunk 3]
```diff
@@ -37,18 +37,27 @@ Eventually these tests will be replaced with BMO's own
...
 * **/test-ubuntu-integration-release-1-5** runs ansible integration tests with CAPM3
   release-1.5 branch and BMO **release-0.4 branch** on Ubuntu
 
+**Release-0.3 branch:**
+
+* **/test-centos-e2e-integration-release-1-4** runs CAPM3 e2e integration tests
+  with CAPM3 release-1.4 branch and BMO **release-0.3 branch** on Centos
```
Comment (tuminoid) [inline README.md:57]:
```suggestion
  with CAPM3 release-1.4 branch and BMO **release-0.3** branch on Centos
```

File: README.md [hunk 4]
```diff
@@ -37,18 +37,27 @@ Eventually these tests will be replaced with BMO's own
...
+* **/test-ubuntu-integration-release-1-4** runs ansible integration tests with CAPM3
+  release-1.4 branch and BMO **release-0.3 branch** on Ubuntu
```
Comment (tuminoid) [inline README.md:59]:
```suggestion
  release-1.4 branch and BMO **release-0.3** branch on Ubuntu
```

**Review by tuminoid — COMMENTED** [review]
Nit: consistent bolding compared to main branch

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**